src/layers/group.rs

```diff
@@ -81,9 +81,12 @@ impl GroupLayerData {
 map_wrapper!(GroupLayer => GroupLayerData);
 
 impl<'map> GroupLayer<'map> {
+    /// Returns an iterator over the layers present in this group, ordered in the same way as in the
+    /// TMX file it was loaded from.
```

Comment:
I'm wondering if it's really helpful to document that the order matches that of the file. It should be expected behavior. Maybe at least remove the "TMX" since otherwise we have to remember to remove it when adding support for the JSON format. We also need to remember to reword it when this library allows dynamic creation of maps.

Alternatively, we'd document the order matches the display order of the layers, so back to front.